noah/src/test/scala/edu/uci/ics/cloudberry/noah/kafka/TestKafka.scala

```diff
@@ -28,12 +28,6 @@ class TestKafka extends Specification with Mockito {
       argument.getValue.isInstanceOf[ProducerRecord[String, String]] must(beTrue)
     }
 
-    "close should be called" in {
```

Comment:
Not necessary the case. The better practice is to address the comment while you fixing the code. Before you upload a new patch you should check all the comments have been `done` or replied.